EventFilter/HcalRawToDigi/src/HcalDataFrameFilter.cc

```diff
@@ -92,6 +129,29 @@ ZDCDigiCollection HcalDataFrameFilter::filter(const ZDCDigiCollection& incol, Hc
   return output;
 }
 
+QIE10DigiCollection HcalDataFrameFilter::filter(const QIE10DigiCollection& incol, HcalUnpackerReport& r) {
+  QIE10DigiCollection output(incol.samples());
+  for (QIE10DigiCollection::const_iterator i=incol.begin(); i!=incol.end(); i++) {
+    if (!HcalDataFrameFilter_impl::checkQIE10(*i,requireCapid_,requireDVER_))
+      r.countBadQualityDigi(i->id());
+    // Never exclude QIE10 digis as their absence would be
+    // treated as a digi with zero charged deposited in that channel
+    output.push_back(*i);
```

Comment:
So, it sounds like there is some hack going on with treatment of QIE10 which violates the pattern for other frame types